bugbug/model.py

```diff
@@ -372,8 +372,14 @@ def train(self, importance_cutoff=0.15, limit=None):
         # Extract features from the items.
         X = self.extraction_pipeline.transform(X_gen)
 
-        # Calculate labels.
         y = np.array(y)
+
+        is_multilabel = isinstance(y[0], np.ndarray)
+        is_binary = len(self.class_names) == 2
+
+        # Calculate labels.
+        if is_multilabel:
+            self.le = LabelBinarizer()
```

Comment:
Why do we need to change the encoder here instead of using it in the model itself in the first place?

Reply:
I thought it would be wiser to fix the error for other models too expect `bugtype` that are multi-label.

Reply:
I think, `bugtype` is the only one with multi-label. Changing the encoder in the training method can be tricky and make the codebase more complex since it is initially defined in the contractor.

Reply:
Oh, Yes. I will revert and change it for the model itself.  Thank you.

Reply:
One alternative solution would be to define `LabelEncoder` in the training method instead of in the constructor, which will make it less confusing.